docs/blog/10-metadata-governance.md

```
@@ -133,12 +164,20 @@ Default credentials:
 ```
 trino
 ```
+Expected output:
+```text
+Example rendered as shown.
+```
 
 **Description:**
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)